SleepEngine/Engine/Jobs/JobAffinity.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+BEGIN_NAMESPACE_SLEEP
+
+enum class JobAffinity
+{
+    Render, // executed only on the render thread
+    Generic, // executed on any of worker threads
+    Count,
+    Invalid

```

Comment:
I propose to make 'Invalid' = 0 since in the most cases uninitialized enum equals 0

Reply:
this way count will not work)

Reply:
I guess thats why its always last